### PR TITLE
Add missing attribute

### PR DESCRIPTION
**Pull request by faustinoaq**
### Description of the Change

**Bux fix** add missing attribute on input field used by delete button

Also `main.bundle.js` and `amber.min.js` were recreated

### Alternate Designs

Not applicable

### Benefits

Fix broken user experience

### Possible Drawbacks

Not applicable (bug fix)

### CODE REVIEW

**Comment by faustinoaq** [comment]
amber

File: src/amber/cli/templates/scaffold/view/src/views/{{name}}/show.ecr.ecr [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <p><%="<"%>%= <%= @name %>.<%= field.name %><%= field.reference? ? ".id" : "" %> %></p>
 <% end -%>
 <p>
-  <%="<"%>%= link_to("back", "/<%= Inflector.pluralize(@name) %>s", class: "btn btn-success btn-xs") -%>
```
Comment (elorest) [inline src/amber/cli/templates/scaffold/view/src/views/{{name}}/show.ecr.ecr]: What are these changes about and how are they related to delete?

File: assets/js/amber.js [hunk 2]
```diff
@@ -242,6 +242,7 @@ document.addEventListener("DOMContentLoaded", function () {
                 form.setAttribute("action", element.getAttribute("href"));
                 form.setAttribute("method", "POST");
                 input.setAttribute("type", "hidden");
+                input.setAttribute("name", "_method");
```
Comment (elorest) [inline assets/js/amber.js]: Oh man. You changed the code from jquery to this so fast last night I just assumed it had everything and didn't look at the details. Sorry.

Comment (faustinoaq) [reply assets/js/amber.js]: Yeah, my bad, I missed this line, I guess I was tired too :sweat:

File: src/amber/cli/templates/scaffold/view/src/views/{{name}}/show.ecr.ecr (hunk 1 above)
Comment (faustinoaq) [reply src/amber/cli/templates/scaffold/view/src/views/{{name}}/show.ecr.ecr]: Oh, ok Let'me remove this an open another PR :+1:

**Comment by faustinoaq** [comment]
> I don't see how the show/back links have anything to do with this.

Oh, ok Let'me remove this an open another PR :+1:

**Pull request merged**